rfcs/text/0000-bundle-definition.md

```diff
@@ -0,0 +1,248 @@
+
+There are currently no documented definitions for what a "bundle" is in the HCA DCP, including no definitions of what
+they are, what they contain, or who they support. Lack of clarity causes confusion for data consumers (and DCP
+developers) about how the data and metadata in the DCP are structured and organized. At the June 2019 DCP F2F meeting in
+Cambridge, a breakout session was held to discuss "What is a bundle?", and a lot of useful discussion occurred. This RFC
+will formally document what a bundle is, anchoring the definition on how bundles support HCA DCP data consumers. This
+RFC will also describe new bundle types that could be supported by the HCA DCP.
+
+### User Stories
+
+1. As a data consumer who develops data processing tools (*e.g.* a sequence read aligner), I would like to browse,
+   query, and get primary data from the DSS so that I can use it as testing data during development.
+
+1. As a data consumer who develops analysis tools (*e.g.* a clustering algorithm), I would like to browse, query, and
+   get alignment data from the DSS so that I can use it as testing data during development.
+
+1. As a data consumer who is a computational biologist, I would like to get expression matrices from the DSS so that I
+   can compare against data I already have.
+
+1. As a data consumer who is a biologist, I would like to get expression matrices from the DSS so that I can see the
+   expression profile of &lt;my gene of interest&gt; in &lt;my cell type(s) or organ(s) of interest&gt;.
+
+1. As a data consumer who is a computational biologist, I would like to know what reference data was used to generate
+   processed data in the DSS and I would like to be able to get it for myself so that I can process/analyze my own data
+   with the same references as HCA DCP data.
+
+1. As a tool developer, I would like to identify and get raw data in the DSS so that I can run my data processing and
+   analysis tools.
+
+1. As a tool developer, I would like to identify and get alignment results and count matrices in order to produce
+   expression matrices.
+
+1. As an external user of the DSS who is developing analysis tools, I would like to browse, query, and get raw and
+   processed data based on some rough criteria so that I can understand what the data are and develop my tool.
+
+1. As an Azul or Query Service developer I want to be able to distinguish between different types of bundles in order to
+   know which bundle to index, link, etc. or to provide bundle types to users as a search criterion.
+
+1. As a computational biologist, I would like to use the same references that the HCA DCP uses in their analysis.
+
+## Detailed Design
+
+### What is a bundle?
+
+A bundle is a unit of data organization, roughly equivalent to a folder or directory. In the HCA DCP, a bundle is
+defined as either a transactional grouping of files that belong together for scientific reasons, or a useful association
+between a set of files for utility purposes.
```

Comment:
We should consider mentioning that bundles are the units of discoverability in the data-store.

Reply:
Added an extra sentence for this, thanks.